great-idea/index.html

```diff
@@ -20,4 +20,96 @@
       
     
   </body>
-</html>
\ No newline at end of file
+</html>
+
+<div class="container">
+
+  <header>
+    <nav>
+      <a href="services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </nav>
+    <a href="index.html">
+              <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> </a>
```

Comment:
This looks great but I would suggest you use indentation appropriately,  for example:
```
<a> 
    <img />
</a>
```
Also, don't forget to use self-closing tag for your `img` element, in a world of software engineers, these things are known as "clean code practices". Typically, when you start working as an actual software engineer, you are expected to write clean and flawless code. So this is something for you to keep in mind.